64.py

```diff
@@ -0,0 +1,17 @@
+class Solution:
+    def minPathSum(self, grid: List[List[int]]) -> int:
+        m, n = len(grid), len(grid[0])
+        dp = [[0] * n for _ in range(m)]
+
+        for i in range(m):
```

Comment:
LGTM. You can also solve this problem using a top-down approach with memoization (i.e., `@lru_cache(None)`).